From c52ce983b05938b220844fc39e749af6e4a5782c Mon Sep 17 00:00:00 2001
From: Krishnamurthy D V <kmurthy@netlogicmicro.com>
Date: Tue, 29 Jun 2010 15:53:08 +0530
Subject: [PATCH 078/565] Generic IRQ flow handler support

Generic IRQ flow handler support Removed do_phnx_IRQ() - not required anymore

Based on Broadcom SDK 2.3.

Signed-off-by: Krishnamurthy D V <kmurthy@netlogicmicro.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/netlogic/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/netlogic/Kconfig b/arch/mips/netlogic/Kconfig
index ec5771d..837212d 100644
--- a/arch/mips/netlogic/Kconfig
+++ b/arch/mips/netlogic/Kconfig
@@ -2,6 +2,7 @@ config NLM_COMMON
 	bool 
 
 config NLM_XLR
+	select GENERIC_HARDIRQS_NO__DO_IRQ
 	bool
 
 config NLM_XLP
-- 
1.8.4.93.g57e4c17

